### DISCUSSION TITLE
Can you run an IDH node on a 2.3.90 environment?

### DISCUSSION THREAD

**Discussion by tomjones1977** [Security-Onion-Solutions/securityonion#7539] [Unsupported Versions] [answered]
The IDH node looks really interesting, but my main grid is on 2.3.90.  Can you install an IDH node on 2.3.110 and connect it to the 2.3.90 grid or will the grid need upgrading first?

**Comment by dougburks** [comment, marked as answer]
We only support IDH in 2.3.110 grids.
